### DISCUSSION TITLE
how to be a part of team?

### DISCUSSION THREAD

**Discussion by nullptr03** [ROM-builders/temporary#21572] [General]
Hi, i want to be a part of team, and how, so i can be a part of your team?

**Comment by Apon77** [comment]
Please follow #team and #learn instruction from telegram group

**Comment by nullptr03** [comment]
here the script repo: https://github.com/nullptr03/temporary1 
and this is the succesfully task: https://github.com/ROM-builders/temporary/pull/21563/checks
sorry if i can't enclose the accurate proof link🙏

**Comment by hipexscape** [comment]
please learn git 
you dont even know simple commands like rebase, cherry-pick, etc
